frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -57,6 +57,24 @@ interface NewRunParametersProps {
   handleParameterChange?: (parameters: RuntimeParameters) => void;
 }
 
+const inputConverter = (paramStr: string, paramTypeIdx: number) => {
```

Comment:
We don't need to define a `const` in this case, you can just define it as regular function:

`function convertInputType(paramStr: string, paramTypeIdx: number): any`

Reply:
Also: Why is parameter type an index? Can we use type `ParameterType_ParameterTypeEnum` instead of `number`?

Reply:
We can either use `number` or `string` for this input. It will lead different `case` under the `switch` in this function statement.

Reply:
So what is the error if you specify it as an enum parameter type? `function convertInputType(paramStr: string, paramType: ParameterType_ParameterTypeEnum)`